internal/setup/fetch.go

```diff
@@ -38,7 +38,7 @@ func FetchRelease(options *FetchOptions) (*Release, error) {
 		cacheDir = cache.DefaultDir("chisel")
 	}
 
-	dirName := filepath.Join(cacheDir, "releases", options.Label + "-" + options.Version)
+	dirName := filepath.Join(cacheDir, "releases", options.Label+"-"+options.Version)
```

Comment:
Not sure what's happened here, but I think it might be cleaner to put these formatting fixes in another PR

Reply:
Whoops. Thanks for the catch. That was just `go fmt` fail. :-) Ditched from this PR.